ivy/functional/frontends/torch/tensor.py

```diff
@@ -419,6 +419,12 @@ def tril(self, diagonal=0):
     def index_select(self, dim, index):
         return torch_frontend.index_select(self._ivy_array, dim, index)
 
+    @with_unsupported_dtypes({"1.11.0 and below": ("bfloat16", "float16")}, "torch")
+    def clamp(self, min=None, max=None, *, out=None):
+        if min is not None and max is not None and min > max:
+            return torch_frontend.tensor(ivy.minimum(ivy.array(self._ivy_array), max))
```

Comment:
you should be able to use the previous `full_like` because it's mentioned that if min > max, all inputs are made to the value of max